t/v3-body.t

```diff
@@ -10,6 +10,20 @@ post '/test' => sub {
   },
   'test';
 
+post '/test/optional/explicitly' => sub {
+  my $c = shift;
+  $c->openapi->valid_input or return;
+  $c->render(json => undef, status => 200);
```

Comment:
Using “json” instead of “openapi” completely bypass the output validator. Same below.

Reply:
Thanks for your review! :smile: